### ISSUE TITLE
Broken file link in `run_corpora_and_vector_spaces` tutorial

### ISSUE THREAD

**Issue by piskvorky** [RaRe-Technologies/gensim#2872]
#### Problem description

The `run_corpora_and_vector_spaces.ipynb` tutorial depends on a file on the web, and that file is missing.

#### Steps/code/corpus to reproduce

See https://groups.google.com/g/gensim/c/nX4lc8j0ZO0

#### Versions

Please provide the output of:

```python
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import gensim; print("gensim", gensim.__version__)
from gensim.models import word2vec;print("FAST_VERSION", word2vec.FAST_VERSION)
```

Unknown (probably any).

**Comment by piskvorky** [comment]
I put the file back for now.

Not sure whether we need that file on the web (as opposed to creating it directly within the notebook – the file is trivial).

The download link in the notebook seems broken too, at least when viewed on github:
![Screenshot (21)](https://user-images.githubusercontent.com/610412/86527739-3b793500-bea2-11ea-966a-e9689f1fc4ae.png)

**Comment by piskvorky** [comment]
Fixed by #3148.

**Comment by raffaem** [comment]
The research paper here:
https://nlp.fi.muni.cz/~xrehurek/nips/rehurek_nips.pdf
is also on 404

**Comment by piskvorky** [comment]
You're right :( Could you update that link to https://arxiv.org/abs/1102.5597 please?

**Comment by raffaem** [comment]
On this webpage:

https://radimrehurek.com/gensim/wiki.html#wiki

The Hoffman reference is broken both in pdf and in code

**Comment by piskvorky** [comment]
Is that the one fixed in #3133?

@mpenkov let's plan the next Gensim release, the fixes and reports are starting to overlap :)
